### PR TITLE
V2 > Loader component

### PR DESCRIPTION
**Pull request by juanfra**
🎫 https://moderntribe.atlassian.net/browse/ET-866
📹 https://www.loom.com/share/6313010c47854c27891e2e5b8850e6d8

### CODE REVIEW

File: src/views/v2/components/loader/loader.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTICLE_LINK_HERE}
+ *
+ * @version TBD
+ */
+
+if ( empty( $text ) ) {
+	$text = $this->get( 'text' ) ?: esc_html__( 'Loading...', 'event-tickets' );
+}
```
Comment (tr1b0t) [inline src/views/v2/components/loader/loader.php:17]: :no_entry_sign: Using short ternaries is not allowed

<sub>hash: 832a1e71d202afd7b14aefa58c71f7d3</sub>

File: src/views/v2/components/loader/loader.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+if ( empty( $loader_classes ) ) {
+	$loader_classes = $this->get( 'classes' ) ?: [];
+}
```
Comment (tr1b0t) [inline src/views/v2/components/loader/loader.php:21]: :no_entry_sign: Using short ternaries is not allowed

<sub>hash: 94b3b610f6f6d5aec42e15a75a3ab0d0</sub>

File: src/views/v2/components/loader/loader.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTICLE_LINK_HERE}
+ *
+ * @version TBD
+ */
+
+if ( empty( $text ) ) {
+	$text = $this->get( 'text' ) ?: esc_html__( 'Loading...', 'event-tickets' );
```
Comment (paulmskim) [inline src/views/v2/components/loader/loader.php:16]: are you using `$text` in this template?

Comment (juanfra) [reply src/views/v2/components/loader/loader.php:16]: Not now for RSVP. But the purpose of _componetizing_ this in `v2/` is to organise everything that is v2 there, so I took the other version as a reference. Where text is a thing.

Comment (juanfra) [reply src/views/v2/components/loader/loader.php:16]: Well, I just re-checked the template and you can forget of what I just said haha. We're not using it on the other template. I'll remove it.

**Pull request merged**